src/Sentry/Platforms/Cocoa/SentrySdk.cs

```diff
@@ -10,11 +10,6 @@ public static partial class SentrySdk
     private static void InitSentryCocoaSdk(SentryOptions options)
     {
         options.LogDebug("Initializing native SDK");
-        // Workaround for https://github.com/xamarin/xamarin-macios/issues/15252
-        ObjCRuntime.Runtime.MarshalManagedException += (_, args) =>
-        {
-            args.ExceptionMode = ObjCRuntime.MarshalManagedExceptionMode.UnwindNativeCode;
-        };
```

Comment:
We don't need to set `ExceptionMode` here as we set it, now, during [compilation](https://github.com/getsentry/sentry-dotnet/blob/191776c9d50c75753d0898cab2660a4dd2e5fdab/src/Sentry.Bindings.Cocoa/buildTransitive/Sentry.Bindings.Cocoa.targets#L12-L13)